Firefox in container use default timezone(UTC) instead of TZ environment variable timezone

Running tests in container with Chrome and Firefox separately as bellow:
(only browser is different between these two command line)
```
// with chrome
docker run \
-e BASE_URL="xxxx" \
-e API_BASE_URL="xxxx" \
-e TZ='Asia/Shanghai' \
testcafe/testcafe chromium /tests/*.test.ts --quarantine-mode --skip-js-errors --disable-page-caching

// with firefox 
docker run \
-e TZ='Asia/Shanghai' \
-e BASE_URL="xxxx" \
-e API_BASE_URL="xxxx" \
testcafe/testcafe firefox /tests/*.test.ts --quarantine-mode --skip-js-errors --disable-page-caching
```
### What is your Test Scenario?
check the text display on the web page is displaying with `Asia/Shanghai` timezone. like '21 Sep 2020 11:49:14 AM'.
 
### What is the Current behavior?
Chrome is displaying '21 Sep 2020 11:49:14 AM'
Firefox   is displaying '21 Sep 2020 03:49:14 AM'
 
### What is the Expected behavior?
displaying time based on given timezone(TZ)
Chrome is displaying '21 Sep 2020 11:49:14 AM'
Firefox   is displaying '21 Sep 2020 11:49:14 AM'
 
<summary>Your complete test report:</summary>

```
22-Sep-2020 23:09:40 | 1) AssertionError: expected '21 Sep 2020 03:49:14 AM' to deeply equal
22-Sep-2020 23:09:40 | '21 Sep 2020 11:49:14 AM'
22-Sep-2020 23:09:40 |  
22-Sep-2020 23:09:40 | Browser: Firefox 78.0 / Linux 0.0
```

### Steps to Reproduce
1. start web server
2. run commands to run test with testcafe docker container (docker image testcafe/testcafe) 
3. Test run with Firefox failed
 
### Your Environment details:
 
* testcafe version:                   1.8.4
* node.js version:                    12.4.0
* command-line arguments:  as beginning
* browser name and version: Chrome  and Firefox 78.0 / Linux 0.0
* platform and version:          testcafe docker container

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -14,7 +14,7 @@ ENV ALPINE_REPOS="\
 
 RUN apk --no-cache $ALPINE_REPOS upgrade && \
  apk --no-cache $ALPINE_REPOS add \
- libevent nodejs npm chromium firefox-esr xwininfo xvfb dbus eudev ttf-freefont fluxbox procps
+ libevent nodejs npm chromium firefox-esr xwininfo xvfb dbus eudev ttf-freefont fluxbox procps tzdata
 
 RUN npm install -g /opt/testcafe/${packageId} && \
  npm cache clean --force && \

